src/components/shared/Avatar/Avatar.tsx

```diff
@@ -21,7 +21,7 @@ export interface Props {
   notSet?: boolean;
 
   /** Icon name to use for placeholder */
-  placeholderIcon: string;
+  placeholderIcon?: string;
```

Comment:
Setting this to optional with a default because it's only needed if `notSet` is true.